tests/python/frontend/caffe/test_forward.py

```diff
@@ -586,6 +586,27 @@ def test_forward_LRN():
     _test_lrn(data, local_size=3, alpha=2.0, beta=0.5, k=2.0)
 
 
+#######################################################################
+# Permute
+# -----------
```

Comment:
Alignment

Reply:
@FrozenGene 
Sorry, can you specifically point out what's wrong with the code?
There seems to be no problem.